packages/react-ui-kit/styleguide.config.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+const fs = require('fs');
+const path = require('path');
+const docgen = require('react-docgen');
+const packageJson = require('./package.json');
+
+function resolve(...paths) {
+  return fs.realpathSync(path.join(__dirname, ...paths));
+}
+
+module.exports = {
+  title: `Wireapp React Ui Kit v${packageJson.version}`,
+  skipComponentsWithoutExample: true,
+  sections: [
+    {
+      name: 'Forms',
+      components: 'src/Form/**/*.tsx',
+      ignore: [
+        'src/Form/Input.tsx',
+        'src/Form/ShakeBox.tsx',
+        'src/Form/InputBlock.tsx',
+        'src/Form/InputBlock.tsx',
```

Comment:
duplicate

Reply:
done in https://github.com/wireapp/wire-web-packages/pull/2937/commits/3e932ed94ce40189d6cd5e16e580d20f56cc308a